tests/dummy/app/templates/migration-guides.hbs

```diff
@@ -67,7 +67,7 @@
       <p>
         Version 0.3.0 includes major column resizing changes, but with
         <code>canAutoResize</code> defaulting to <code>true</code>.  We
-        recommend skipping this version and upgrading directly to 0.4.0; see
+        recommend skipping this version and upgrading direcly to 0.4.0; see
```

Comment:
No, no, vim couldn't have been responsible for this mistake. Maybe I was trying out emacs at the time.